e2e/playwright/actions/src/tests/create-library.spec.ts

```diff
@@ -0,0 +1,241 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { libraryErrors, ApiClientFactory, getUserState, test } from '@alfresco/playwright-shared';
+import { SiteBodyCreate } from '@alfresco/js-api';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('Create Libraries ', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let randomLibraryName: string;
+  let randomLibraryId: string;
+  let randomLibraryDescription: string;
+  const libraryDialogTitle = 'Create Library';
+  const libraryNameLebel = 'Name *';
+  const libraryIdLebel = 'Library ID *';
+  const libraryDescriptionLebel = 'Description';
+  const publicVisibility = 'Public';
+  const moderatedVisibility = 'Moderated';
+  const privateVisibility = 'Private';
+  const deleteAction = 'Delete';
+
+  const commonLibraryName = `playwright-library-${(Math.random() + 1).toString(36).substring(6)}`;
```

Comment:
we already have random utility that is safe:

```ts
import { Utils } from '@alfresco/aca-testing-shared';
const username = `user-${Utils.random()}`;
```

Reply:
doesn't act-testing-shared is a protractor part? You mean, that we better create similar for playwright use ?